### DISCUSSION TITLE
Continuous testing config properties in pom.xml not working anymore in Quarkus 3.13.0

### DISCUSSION THREAD

**Discussion by famod** [quarkusio/quarkus#42326] [Q&A] [answered]
Hey all (especially @radcortez)!

With Quarkus 3.13.0 (or 3.13.0.CR1), properties for continuous testing like `quarkus.test.exclude-tags` and `quarkus.test.exclude-engines` cannot be set anymore via pom.xml `<properties>`.
I can however still set them via `mvn ... -Dquarkus.test.exclude-tags=...` etc.

Is this to be expected? I've checked the migration guide and found nothing so far.

Cheers!

[a comment by an automated account is not shown]

**Reply by famod** [reply]
Maybe also /cc @aloubyansky

**Reply by gsmet** [reply]
I would bet I'm the one who broke it: https://github.com/quarkusio/quarkus/pull/41317/commits/00350b7a84a3a1ec0aee49be9e848c84cda2bafd .

But I don't mind you pinging the other folks to fix it :).

**Reply by gsmet** [reply]
What I'm not sure of is which config source is used to extract things from the POM file.

**Reply by famod** [reply]
Hm, good question.

At this point I'm also asking myself if your adjustment of "properties change" is really relevant here because I'm not actually changing something, I'm just running continuous testing without touching anything.

**Reply by gsmet** [reply]
Can you check if reverting my change fixes your issue?

That's the only thing I have in mind that touches this area but... it could also be a more general Config issue. In any case, it would be good to know.

**Reply by famod** [reply]
Will do. Might take some time though.

**Reply by famod** [reply]
@gsmet I can confirm that your commit broke it.

**Reply by famod** [reply, marked as answer]
I filed #42714

**Comment by famod** [comment]
FTR, workaround in quarkus-maven-plugin configuration:
```xml
<jvmArgs>-Dquarkus.test.exclude-engines=${quarkus.test.exclude-engines} -Dquarkus.test.exclude-tags=${quarkus.test.exclude-tags}</jvmArgs>
```
So I'm passing the properties explicitly via -D.
